packages/MainUI/hooks/useTranslation.ts

```diff
@@ -3,8 +3,9 @@ import { TranslateFunction, Translations } from './types';
 import { useCallback, useMemo } from 'react';
 import translations from '@workspaceui/componentlibrary/src/locales';
 
-export const useTranslation = () => {
-  const { language } = useLanguage();
+export const useTranslation = (defaultLanguage = 'en') => {
```

Comment:
should be `en_US` instead of `en`.
most precisely, you should import the `DEFAULT_LANGUAGE` you have declared in `packages/MainUI/contexts/languageProvider.tsx`